requirements.txt

```diff
@@ -12,6 +12,7 @@ mpmath==1.3.0
 networkx==3.2
 numpy==1.26.1
 packaging==23.2
+pandas==2.1.2
```

Comment:
Please make sure to include all of `pandas`'s dependencies here too (`pip freeze` should give you them if you haven't installed anything not in `requirements.txt`)